src/core/lib/surface/init.cc

```diff
@@ -24,6 +24,7 @@
 #include <grpc/fork.h>
 #include <grpc/grpc.h>
 #include <grpc/impl/channel_arg_names.h>
+#include <grpc/support/log.h>
 #include <grpc/support/port_platform.h>
 #include <grpc/support/sync.h>
```

Comment:
We use gpr_log_verbosity_init() in this file. 
So we are adding the include here.